lib/foods/foods.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function() {
+  $(loadFoods)
+})
+
+function loadFoods() {
+  $.get('https://obscure-harbor-85447.herokuapp.com/api/v1/foods')
+  .then(function(foods) {
+    foods.forEach(function(food) {
+      $('.foods-table').append(
+        `<tr class=row${food.id}><td>`+ food.name + '</td>' +
+        '<td>' + food.calories + '</td>' +
+        `<td><input id=${food.id} type='button' class='delete-row' value='Delete'/></td>` +
+        "</tr>"
+      )
+    })
+  })
+}
+
+function deleteFood(e) {
+  $.ajax({
+    type: 'DELETE',
+    url:'https://obscure-harbor-85447.herokuapp.com/api/v1/foods/' + e.target.id,
+    success: $(`tr.row${e.target.id}`).remove()
+  })
+}
+
+$('form').submit(function(e) {
+  e.preventDefault()
+  var food = $('input[name="food"]').val()
+  var calories = $('input[name="calories"]').val()
+  $('.manage-food').each(function() {
+    this.reset()
+  })
+  $('p.food-error').remove()
+
+  $.ajax({
+    type: "POST",
+    url: 'https://obscure-harbor-85447.herokuapp.com/api/v1/foods',
+    data: { food: {
+      name: $.trim(food),
+      calories: $.trim(calories)
+    }},
+    success: function(data){
+      $('.foods-table').append(
+        '<tr><td>'+ data.name + '</td>' +
+        '<td>' + data.calories + '</td>' +
+        `<td><input id=${data.id} type='button' class='delete-row' value='Delete'/></td>` +
+        "</tr>"
+      )
+
+    },
+    error: function(error){
```

Comment:
i like how you're handling the specific errors here.. but could we make this a named function and break this down even further? there's a lot going on here!